sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -361,6 +361,7 @@ def query_items(
         partition_key = kwargs.pop('partition_key', None)
         if partition_key is not None:
             feed_options["partitionKey"] = self._set_partition_key(partition_key)
+            kwargs["containerProperties"] = self._get_properties
```

Comment:
Where is this getting awaited?

Reply:
For the async version a work around had to be made. Query items is not an async method and cannot await. We are delaying awaiting the self._get_properties method when we get to an async method that can properly call it and get the information we need to do a prefix query. In this case it will be awaited when we get to _queryfeed in cosmos_client_connection_async.